api/openapi-spec/v1.0.yaml

```
@@ -4246,6 +4246,10 @@ components:
           items:
             $ref: '#/components/schemas/driveItem'
           description: A collection of special drive resources.
+        hasTrashedItems:
```

We should use the '@libre.graph' namespace for our extensions
```suggestion
        '@libre.graph.hasTrashedItems':
```